iOS/Flows/Conversation/Threads/ThreadViewController.swift

```diff
@@ -63,7 +63,11 @@ class ThreadViewController: DiffableCollectionViewController<ConversationSection
         self.messageController = ChatClient.shared.messageController(cid: channelID, messageId: messageID)
         self.conversationController = ChatClient.shared.channelController(for: channelID,
                                                                              messageOrdering: .topToBottom)
-        super.init(with: ThreadCollectionView())
+        let collectionView = ThreadCollectionView()
+        super.init(with: collectionView)
+
+        collectionView.threadLayout.dataSource = self
+        self.messageController.listOrdering = .bottomToTop
```

Comment:
Flipping the order of the replies because that's what the time machine layout expects. I'd like to make ordering more flexible in the future.